pkg/daemon/cleanup/cleanup.go

```diff
@@ -62,6 +70,16 @@ func Deconfigure(cfg *rest.Config, ctx context.Context, configMapNamespace strin
 			cmData, err = servicescm.Parse(cm.Data)
 			return err
 		}
+		// Make a best effort attempt to cordon and drain the node before removing any services. We shouldn't block
```

Comment:
Shouldn't this also be called in the case where we get the node via config map and VersionAnnotation/DesiredVersionAnnotation? 

Maybe add this change into a function and call it here and after line 98

Reply:
Nevermind - I saw your comment below "as applying the version annotation is signaling to WMCO that the node has been completely been configured, and setting spec.unschedulable to false is part of that."